clustering/server/src/test/java/org/wildfly/clustering/server/ServiceLoaderTestCase.java

```diff
@@ -51,10 +51,11 @@ public void load() {
         load(DistributedCacheServiceConfiguratorProvider.class);
         load(LocalGroupServiceConfiguratorProvider.class);
         load(LocalCacheServiceConfiguratorProvider.class);
+        load(SerializationContextInitializer.class);
     }
 
     private static <T> void load(Class<T> targetClass) {
         ServiceLoader.load(targetClass, ServiceLoaderTestCase.class.getClassLoader())
-                .forEach(object -> LOGGER.trace("\t" + object.getClass().getName()));
+                .forEach(object -> System.out.println("\t" + object.getClass().getName()));
```

Comment:
May want to get rid of this System.out running loose here in this test case.

Reply:
Yeah...